src/portfft/dispatcher/workgroup_dispatcher.hpp

```diff
@@ -85,10 +85,10 @@ IdxGlobal get_global_size_workgroup(IdxGlobal n_transforms, Idx subgroup_size, I
  * `SpecConstComplexStorage`) is split, this is just the real part of data.
  * @param output accessor or pointer to global memory for output data. If complex storage (from
  * `SpecConstComplexStorage`) is split, this is just the real part of data.
- * @param input accessor or pointer to global memory containing imaginary part of the input data if complex storage
- * (from `SpecConstComplexStorage`) is split. Otherwise unused.
- * @param output accessor or pointer to global memory containing imaginary part of the input data if complex storage
+ * @param input_imag accessor or pointer to global memory containing imaginary part of the input data if complex storage
```

Comment:
Don't we always pass the pointer to our `*_impl` functions, even when it is an accessor, by taking the address of the of first element ?